### DISCUSSION TITLE
Should porter list and porter installations list have the same output

### DISCUSSION THREAD

**Discussion by iennae** [getporter/porter#1323] [Q&A]
As I'm updating the quickstart, and thinking about the different definitions of concepts is there a difference between

* installed bundles
* installations

One of the examples that I read has you installing naming the installation demo instead of accepting the default bundle name. This means you can have multiple installations of a bundle with different names. The default view of both porter install list and porter installations list is the same:

```
➜  ~ porter installations list
NAME       CREATED          MODIFIED         LAST ACTION   LAST STATUS
HELLO      25 minutes ago   25 minutes ago   install       succeeded
```

What if one of these commands actually output something like

NAME BUNDLE NAME CREATED          MODIFIED         LAST ACTION   LAST STATUS

This would also let you be able to see the difference if multiple bundles had the same installation name. 

I haven't gone through the source code or spec to actually see whether this would be ok, but wanted to get this out of my head so I can finish up the quickstart update.

**Comment by carolynvs** [comment]
I wrote up a really long nice reply with suggested formatting changes to commands and... poof lost. 😭 

> This would also let you be able to see the difference if multiple bundles had the same installation name.

You cannot reuse installation names, they need to be unique. I think we have an open issue to prevent someone from say installing bundle X using a name, then installing another different bundle Y with the same name. Right now porter will let you do that but what happens is that it ends up screwing up the previous installation of bundle X.

To answer the main question, yes `porter list` and `porter installations list` should have the same output because `porter list` is an alias for `porter installations list`.

But that's not helpful and you asked some good questions about what information would be helpful to show people! So looking at what we could do to improve:

* Update the output of `porter installation list` 
* Update the output of `porter installation show` to display the current bundle version and the bundle version used for each of the actions in History
* Add a command that lets people see more details about each line in that History section. So you could see more about what was done for the first time we ran install for example. We can show the parameters used, outputs generated, and details about the bundle.
* Include the bundle reference, e.g. the value you used with --tag during install, in the `porter installation show` output
